suite-native/module-send/src/screens/SendOutputsScreen.tsx

```diff
@@ -1,314 +1,104 @@
-                },
-            }),
-        );
-
-        if (isFulfilled(response)) {
-            dispatch(storeFeeLevels({ feeLevels: response.payload }));
-            navigation.navigate(SendStackRoutes.SendFees, {
-                accountKey,
-                tokenContract,
-            });
+export const SendOutputsScreen = ({
+    route: { params },
+}: StackProps<SendStackParamList, SendStackRoutes.SendOutputs>) => {
+    const { accountKey, tokenContract } = params;
+    const sendForm = useSendForm(accountKey, tokenContract);
+    const { totalSelectedAmount, selectedUtxos } = useUtxoSelection();
 
-            return;
-        }
-    });
+    if (!sendForm) {
+        return null;
+    }
+    const { form, isValid, isSubmitting, handleNavigateToReviewScreen, amount, network } = sendForm;
+    const isMissingUtxos =
+        selectedUtxos.length > 0 && amount && Number(totalSelectedAmount) < Number(amount);
 
     return (
         <Screen
             header={
                 <AccountBalanceScreenHeader accountKey={accountKey} tokenContract={tokenContract} />
             }
             footer={
-                isValid && (
-                    <Animated.View entering={SlideInDown} exiting={SlideOutDown}>
-                        <ScreenFooterGradient />
-                        <Box style={applyStyle(screenFooterStyle)}>
-                            <Button
-                                accessibilityRole="button"
-                                accessibilityLabel="validate send form"
-                                testID="@send/form-submit-button"
-                                onPress={handleNavigateToReviewScreen}
-                                isDisabled={isSubmitting}
-                            >
-                                <Translation id="generic.buttons.continue" />
-                            </Button>
-                        </Box>
-                    </Animated.View>
+                isMissingUtxos ? (
+                    <HStack padding="sp16">
+                        <SelectAdditionalCoinsWarning />
+                    </HStack>
+                ) : (
+                    isValid && (
+                        <Footer
+                            isSubmitting={isSubmitting}
+                            handleNavigateToReviewScreen={handleNavigateToReviewScreen}
+                        />
+                    )
                 )
             }
             focusedInputBottomOffset={148} // space below the main amount input + button height with vertical margin
         >
             <>
                 <AccountDetailsCard accountKey={accountKey} tokenContract={tokenContract} />
+
                 <Box marginTop="sp32">
                     <Form form={form}>
                         <SendOutputFields accountKey={accountKey} />
+                        {network?.networkType === 'bitcoin' && (
+                            <HStack justifyContent="center" marginTop="sp24">
```

Comment:
`HStack` with a single child has no benefit over simple `Box`.
```suggestion
                            <Box justifyContent="center" marginTop="sp24">
```

Reply:
ca79ab1df38447a6c4cfe999a927ae41a793df59